docs/release-notes/8.9.asciidoc

```diff
@@ -0,0 +1,58 @@
+==== Breaking changes
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
+
+:pull: https://github.com/elastic/kibana/pull/
+
+There are no breaking changes in 8.9.0.
+
+//end::breaking-changes[]
+
+[discrete]
+[[deprecations-8.9.0]]
+==== Deprecations
+* Removes the option to use the legacy navigation menu ({pull}158094[#158094]).
+
+[discrete]
+[[features-8.9.0]]
+==== New Features
+* Allows you to install the Cloud Security Posture Management (CSPM) integration via CloudFormation ({pull}159994[#159994]).
+* Creates a new dashboard, Cloud Native Vulnerability Management, that provides an overview of vulnerabilities on your cloud hosts ({pull}159699[#159699]).
+* Allows you to group vulnerabilities by resource (host) on the Vulnerabilities Findings page, and creates a Resource flyout that displays detailed vulnerability findings for individual hosts ({pull}159873[#159873], {pull}158987[#158987]).
+* Adds a new custom dashboard, "Detection rule monitoring" ({pull}159875[#159875]).
+* Allows you to anonymize event field values sent to AI Assistant ({pull}159857[#159857]).
+* Adds a *Chat* button that opens AI Assistant to the alert details flyout ({pull}159633[#159633]).
+* Updates AI Assistant to let you create and delete custom system prompts and default conversations ({pull}159365[#159365]).
+* Adds a tour that introduces the EQL missing events feature ({pull}158863[#158863]).
+* Lets you add tags to alerts ({pull}157786[#157786]).
+* Adds a tooltip to snoozed rules showing exactly when alerting will resume ({pull}157407[#157407]).
+
+[discrete]
+[[enhancements-8.9.0]]
+==== Enhancements
+* Adds a *Last response* dropdown menu to the Rules table that allows you to filter rules by the status of their last execution ("Succeeded", "Warning", or "Failed") ({pull}159865[#159865]).
+* Creates a Lens dashboard for monitoring the use of tokens by AI Assistant ({pull}159075[#159075]).
+* Creates a connector for D3 Security ({pull}158569[#158569]).
+* Improves the interface for adding and updating Elastic prebuilt rules ({pull}158450[#158450]).
+* Shows a rule's actions on its details flyout ({pull}158189[#158189]).
+* Allows you to add Lens visualizations to cases from the visualization's *More actions* menu ({pull}154918[#154918]).
+
+[discrete]
+[[bug-fixes-8.9.0]]
+==== Bug fixes
+* Fixes a bug that restricted the number of cloud accounts which could appear on the Cloud Security Posture dashboard to 10 ({pull}157233[#157233]).
+* Fixes a bug that allowed you to save a rule that has an alert filter missing a query ({pull}159690[#159690]).
+* Fixes unexpected filtering behavior on the Alert page. When you select a filter that excludes all alerts, an empty table now appears ({pull}160374[#160374]).
```

Comment:
Currently, it is not very clear, what is the intention of this fix. Please feel free to suggest more changes to below suggestion. 


```suggestion
* Fixes unexpected filtering behavior on the Alert page. In previous version when you selected a filter value say `open` for the alert `Status` and there are no open alerts available, system would ignore `open` option and will show rest of the alerts even if they are `closed`. New behavior does not ignore `open` option and will show an empty table if there are not `open` alerts available ({pull}160374[#160374]).
```

Reply:
Hey, @logeekal! Appreciate the review. We generally aim to keep release notes summaries as succint as possible and then include a PR link so users can learn more if they're interested. Since your suggested revision highlights the before and after behavior, we're thinking about modifying our summary to something like:

* Fixes unexpected filtering behavior on the Alert page. Now, when you select a filter that excludes all alerts, an empty table now appears as expected ({pull}160374[#160374]).

Let us know what you think - thank you!

Reply:
@nastasha-solomon , thanks for your response. That makes sense.  It is good know that PR will linked as well for the customers.